homeassistant/components/prusalink/sensor.py

```diff
@@ -79,6 +79,45 @@ class PrusaLinkSensorEntityDescription(
             value_fn=lambda data: cast(float, data["telemetry"]["temp-nozzle"]),
             entity_registry_enabled_default=False,
         ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.telemetry.temp-bed.target",
+            name="Target Heatbed Temperature",
```

Comment:
Please only capitalize the first word in the name.

https://developers.home-assistant.io/docs/core/entity?_highlight=entity&_highlight=nam#entity-naming

Reply:
I fixed the naming scheme for the new entities and the existing ones with a name containing more than one word.